harness/attributes/docker-base.yml

```diff
@@ -63,13 +63,39 @@ attributes:
       resources:
         memory: "1024Mi"
     redis:
-      image: redis:5-alpine
+      image: redis:6-alpine
+      options:
+        # Handle many smaller keys
+        activedefrag: 'yes'
+        # 1Gi - 1024*1024*1024 bytes
+        maxmemory: '1073741824'
+        # Evict any least recently used key even if they don't have a TTL
+        maxmemory-policy: allkeys-lru
+        # Save snapshots every X changes have happened within Y seconds (these are the defaults in redis.conf)
+        save:
+          - 3600 1
+          - 300 100
+          - 60 10000
       resources:
-        memory: "256Mi"
+        # 1.5 * maxmemory to allow copy on write snapshots
+        memory: "1.5Gi"
```

Comment:
quite high defaults for projects running 1-2 nodes

Reply:
We can decrease the maxmemory to allow this to change too?

Reply:
yep, is a good PR in general